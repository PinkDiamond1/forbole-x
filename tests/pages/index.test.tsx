import React from 'react'
import renderer from 'react-test-renderer'
import Home from '../../pages/index'

jest.mock('../../components/Layout', () => 'Layout')
jest.mock('../../components/AccountCard', () => 'AccountCard')
jest.mock('../../components/AssetDistributionChart', () => 'AssetDistributionChart')
jest.mock('../../components/WalletBalanceChart', () => 'WalletBalanceChart')

describe('page: Home', () => {
  it('renders correctly', () => {
    const component = renderer.create(<Home />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})

afterEach(() => {
  jest.clearAllMocks()
})
