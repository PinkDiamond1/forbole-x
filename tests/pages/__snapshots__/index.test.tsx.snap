// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`page: Home renders correctly 1`] = `
<Layout
  activeItem="/"
  passwordRequired={true}
>
  <div
    className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-md-6"
    >
      <WalletBalanceChart />
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-md-6"
    >
      <AssetDistributionChart />
    </div>
  </div>
  <div
    className="MuiBox-root MuiBox-root-1"
  >
    <h4
      className="MuiTypography-root MuiTypography-h4 MuiTypography-gutterBottom"
    >
      common:accounts
    </h4>
  </div>
  <div
    className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-md-6 MuiGrid-grid-lg-4"
    >
      <AccountCard />
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-md-6 MuiGrid-grid-lg-4"
    >
      <AccountCard />
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-md-6 MuiGrid-grid-lg-4"
    >
      <AccountCard />
    </div>
  </div>
</Layout>
`;
